src/REISE.jl

```diff
@@ -9,9 +9,9 @@ import Requires
 import SparseArrays: sparse, SparseMatrixCSC
 
 
-include("types.jl")         # Defines Case, Results, Storage,
+include("types.jl")         # Defines Case, Results, Storage, Flexibility,
```

Comment:
Yeah, I can rename that

Reply:
We are kind of using 'flexibility' to refer 'demand flexibility' both in variable names and doc strings. Not sure we should be more specific just in case we might define other flexibilities in the future.

Reply:
Yeah, that's a good point. I'll be sure change `Flexibility` to `DemandFlexibility` in `types.jl` (which is where that notation currently originates). I'll make sure other doc strings are updated similarly

Reply:
Done.